src/shared/components/app/footer.tsx

```diff
@@ -1,6 +1,6 @@
 import { Component } from "inferno";
 import { NavLink } from "inferno-router";
-import { GetSiteResponse } from "lemmy-js-client";
+import { GetSiteResponse } from "lemmy-js-client/dist/types/GetSiteResponse";
```

Comment:
Can the types not be imported from just `lemmy-js-client`?

Reply:
@dessalines I made a PR for the js client that should get rid of the need for these individual imports: https://github.com/LemmyNet/lemmy-js-client/pull/118.

Reply:
Thanks, I'll make this a draft until I fix up the imports.